packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -14,19 +14,24 @@ import 'timeline_protocol.dart';
 
 class FramesBarChart extends CoreElement {
   FramesBarChart(TimelineController timelineController)
-      : super('div', classes: 'timeline-frames section-border') {
+      : super('div', classes: 'timeline-frames') {
     layoutHorizontal();
     element.style
       ..alignItems = 'flex-end'
       ..height = '${chartHeight}px'
       ..paddingTop = '${topPadding}px';
 
-    timelineController.onFrameAdded.listen((TimelineFrame frame) {
-      if (frameUIgraph == null) {
-        frameUIgraph = PlotlyDivGraph(this, frame, false); // Process chunks
-        add(frameUIgraph);
-      }
+    frameUIgraph = PlotlyDivGraph(this, timelineController); // Process chunks
```

Comment:
Is this comment accurate? Are chunks being processed on initialization?

Reply:
Good catch, cut and paste job its delayed from this this code now.